commit 15f5e3a14b9df029237da25060270178e4f0d96e (refs/changes/18/4618/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-08-02T14:32:42-07:00 (1 year, 2 months ago)
    
    TRITON-656 CNAPI should crash on unhandled exceptions
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/server.js b/lib/server.js
index acd0644..e0cacef 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -23,7 +23,8 @@ function createServer(options) {
     var cnapi = restify.createServer({
         name: 'Compute Node API',
         log: options.log,
-        handleUpgrades: true
+        handleUpgrades: true,
+        handleUncaughtExceptions: false
     });
 
     cnapi.use(restify.requestLogger());
diff --git a/package.json b/package.json
index 223f3e1..3d7a2f3 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.11.2",
+  "version": "1.11.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
